membersrvc/ca/ca.go

```diff
@@ -33,7 +33,7 @@ import (
 
 	"github.com/hyperledger/fabric/core/crypto/primitives"
 	pb "github.com/hyperledger/fabric/membersrvc/protos"
-	_ "github.com/mattn/go-sqlite3"
+	_ "github.com/mattn/go-sqlite3" // TODO: not use a blank import!
```

Comment:
Hi @ghaskins,

1. Yes I manually added the comment and not the tool. So yes, without loss of generality ;-) and under some assumption, it is relatively safe to claim that it was a **human**.

2. It's true that blank imports are generally used to force the side-effects of running the modules func init()... 

As for the second question, I do understand why it is currently there: Specifically in this case I believe the blank import is used for registering the **sqlite3** driver as a database driver without importing any other functions, kind of a:

`sql.Register("sqlite3", &SQLiteDriver{})`

I believe that the coder's original desired side-effects are/were that further (standard) `sql` calls will refer to the registered `sqlite3` driver, e.g., in:

`db, err := sql.Open("sqlite3", filepath.Join(ksPath, ks.node.conf.getKeyStoreFilename()))
`
in [/core/crypto/node_ks.go#L358](https://github.com/JonathanLevi/fabric/blob/ca8135b1a0f95127c96478518628a65a377ec123/core/crypto/node_ks.go#L358)

Which is why I haven't removed the import altogether.

Keeping in mind that the background to this suggested fix is that I wanted to quickly address the go lint message:

`ca/ca.go:35:2: a blank import should be only in a main or test package, or have a comment justifying it`.